contrib/build-push-ceph-container-imgs.sh

```diff
@@ -39,7 +40,8 @@ CONTAINER_BRANCH="${GIT_BRANCH#*/}"
 CONTAINER_SHA=$(git rev-parse --short HEAD)
 TAGGED_HEAD=false # does HEAD is on a tag ?
 DEVEL=${DEVEL:=false}
-OSD_FLAVOR=${FLAVOR:="default"}
+#shaman flavor based on OSD type proporgated by ceph-build
```

Comment:
please note, shaman is just a RESTful API and web UI for accessing the builds and repos. the "flavor" is connected more closely to ceph-build and jenkins.

also, please add a space after `#`.

Reply:
aah, I see. amended PR :)